c99/ulisp.c

```diff
@@ -445,6 +445,15 @@ void delay (int millisecs) {
   delay_on_host(millisecs);
 }
 
+/**
+ * Flush output before reading input
+ */
+EM_JS(void, flush_output, (), {
+  if (globalThis.ulisp && globalThis.ulisp.flushOutput) {
+    globalThis.ulisp.flushOutput();
+  }
+});
+
```

Comment:
Well I tried the "code change suggestion" feature but it's not working for multi-line deletes.

This function is unused so it can be removed.